packages/module/src/Hooks/pagination.ts

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+export interface UseDataViewPaginationProps {
+  page?: number;
+  perPage: number;
+}
```

Comment:
Why is `perPage` required but `page` optional? I think both should be optional IMO.

Reply:
I think we don't want to introduce default values for perPage - they can vary from use case to use case. For page I think it is fine to keep it optional since it is usually 1 at the beginning.